ansible/roles/test/files/tools/loganalyzer/loganalyzer_common_ignore.txt

```diff
@@ -375,6 +375,13 @@ r, ".* INFO audisp-tacplus: Audisp-tacplus: Accounting: user: .*, tty: .*, host:
 # Ignore lldp error log, don't have real impact
 r, ".* ERR container: docker cmd: stop for lldp failed with 500 Server Error.*"
 
+#Ignore Packet Trimming attribute feature error temporarily (no impact)
```

Comment:
@nazariig Could you please check and modify these capability related logs to NOTICE and not ERROR as this will flood many vendor SKUs.

Reply:
@nazariig could you create a PR and update the log? We could skip this temporary fix